packages/gatsby/src/schema/schema.js

```diff
@@ -830,21 +856,24 @@ const resetOverriddenThirdPartyTypeFields = ({ typeComposer }) => {
   })
 }
 
-const processThirdPartyTypeFields = ({ typeComposer, schemaQueryType }) => {
-  resetOverriddenThirdPartyTypeFields({ typeComposer })
-
+const processThirdPartyTypeFields = ({
+  typeComposer,
+  type,
+  schemaQueryType,
+}) => {
   // Fix for types that refer to Query. Thanks Relay Classic!
-  typeComposer.getFieldNames().forEach(fieldName => {
+  const fields = type.getFields()
+  Object.keys(fields).forEach(fieldName => {
     // Remove customization that we could have added via `createResolvers`
     // to make it work with schema rebuilding
-    const field = typeComposer.getField(fieldName)
-    const fieldType = field.type.getTypeName()
+    const fieldType = String(fields[fieldName].type)
```

Comment:
Is this not always a string?

Reply:
No, it's actually an instance of GraphQL type (possibly with `nonNull` and `listOf` wrappers). But those have toString method that gets printed to something like `[GitHub!]`